routes/event.js

```diff
@@ -55,6 +56,48 @@ router.get('/guests/:event_id',isLoggedIn, isAdminUser, function(req, res) {
 	});
 });
 
+router.get('/guests/send/:event_id', isLoggedIn, isAdminUser, function(req, res) {
+	eventMapper.findEventBy_event_id(req.params.event_id, function(err, result) {
+		if (err) {
+			res.send(err);
+		}
+
+		res.render('sendGuestDetails', {result, err: req.flash('err'),succ: req.flash('succ')});
+	});
+});
+
+router.post('/guests/send/:event_id', isLoggedIn, isAdminUser, function(req, res) {
+	eventMapper.findEventBy_event_id(req.params.event_id, function(err, result) {
+		if (!result) {
+			req.flash('err', 'Event not found');
+		}
+		else if (err) {
+			res.send(err);
+		}
+		else
+		{
+			var details = 'The following is a specification of guest dietary restrictions, access requirements and table arrangements for the upcoming event, ' + result.title + ':\n\n';
```

Comment:
Since we have it might be worthwhile adding the date aswell as the title. Isn't necessary though just cosmetic